Agent/entities/Team_entries_en.json

```diff
@@ -0,0 +1,159 @@
+[
```

Comment:
This file should not be commited, since it is dynamically generated. If you delete it locally and rerunn the "exportAgent" script it should not reappear. If it does that is a bug in the "exportAgent" script.

Reply:
will delete